virttest/env_process.py

```diff
@@ -1573,6 +1583,10 @@ def postprocess(test, params, env):
         else:
             _post_hugepages_surp = h.ext_hugepages_surp
 
+    global _kvm_module_handler
+    if _kvm_module_handler:
+        _kvm_module_handler.restore()
```

Comment:
But after you restored you should clean the `_kvm_module_handler` as there are following tests that might use different params and the `restore` would be called multiple times.

Reply:
Fixed with last change.